src/_posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md

```diff
@@ -1,42 +1,132 @@
 ---
 title: Scalingo Scheduler - Run Scheduled Tasks
 nav: Scalingo Scheduler
-modified_at: 2022-10-24 10:00:00
+modified_at: 2023-01-31 17:00:00
 tags: task-scheduling
 index: 1
 ---
 
-Scalingo Scheduler is here to help you run scheduled tasks at regular intervals.
+Scalingo provides a feature called *Scalingo Scheduler* to help you run tasks
+on a regular basis.
 
-The syntax used to describe this interval is the same as in the cron software.
+## About the Scalingo Scheduler
 
-Scalingo Scheduler launches tasks as [one-off containers]({% post_url platform/app/2000-01-01-tasks %}) in detached mode. Therefore the related one-off documentation and their detached mode applies.
+As the name suggests, the Scalingo Scheduler allows to define tasks so that
+they run periodically at fixed times, dates, or intervals. It can be leveraged
+to do a variety of tasks such as sending a newsletter every morning, exporting
+data every hour, checking for new orders every 15 minutes, ...
```

Comment:
nitpick: I don't like the beginning of this paragraph. It does not add any information.

```suggestion
The Scalingo Scheduler allows to define tasks so that
they run periodically at fixed times, dates, or intervals. It can be leveraged
to do a variety of tasks such as sending a newsletter every morning, exporting
data every hour, checking for new orders every 15 minutes, ...
```

Reply:
Fixed in https://github.com/Scalingo/documentation/pull/1915/commits/22a7de15789981fbcc5fd0eeacd80f1081d22d66